clients/src/main/java/org/apache/kafka/common/errors/RebalanceInProgressException.java

```diff
@@ -19,19 +19,7 @@
 public class RebalanceInProgressException extends ApiException {
     private static final long serialVersionUID = 1L;
 
-    public RebalanceInProgressException() {
```

Comment:
Same here. I think we should probably leave these constructors around. They are not really doing any harm.